docs/contributer_guide/install.rst

```diff
@@ -0,0 +1,56 @@
+Developer Installation
+======================
+
+For using ``ngshare``, see :doc:`../user_guide/install`.
+
+Install from GitHub
+----------------
+.. code:: bash
+
+    git clone https://github.com/lxylxy123456/ngshare.git
+    cd ngshare/
+    pip3 install .
+
+Run Installed ngshare
+---------------------
+.. code:: bash
+
+    python3 -m ngshare [arguments]
+
+Run ngshare without Installation
+--------------------------------
+The first line installs pip dependencies
+
+.. code:: bash
+
+    pip3 install tornado jupyterhub sqlalchemy
+    git clone https://github.com/lxylxy123456/ngshare.git
+    cd ngshare/ngshare/
+    python3 ngshare.py [arguments]
+
+Run vngshare
+------------
+vngshare is simply adding some arguments to ngshare.
```

Comment:
I think this could be changed to something like "vnshare can be used by adding some arguments to ngshare" or something along those lines.